README.md

```diff
@@ -68,10 +122,14 @@ The below configuration variables are required for this Connector to operate.  T
 
 VARIABLE | REQUIRED | TYPE | DESCRIPTION
 -------- | -------- | ---- | -----------
-**url** |  required  | string | Complete URL (e.g. http://mybmc.contoso.com:8008)
+**url** |  required  | string | Base URL without port (e.g. http://mybmc.contoso.com)
 **verify_server_cert** |  optional  | boolean | Verify server certificate
-**username** |  required  | string | Username
-**password** |  required  | password | Password
+**username** |  optional  | string | Username
+**password** |  optional  | password | Password
+**client_id** |  optional  | string | Client App ID for (Interactive)/OAuth Authentication
+**client_secret** |  optional  | password | Client Secret for (Interactive)/OAuth Authentication
+**auth_type** |  optional  | string | Authentication Mechanism to Use
+**ports** |  required  | string | Ports to use for SSO and REST calls (comma-separated [First port for REST calls, Second for SSO calls])
```

Comment:
Why not two separate variables if there are two ports? Cleaner, easier to use in code